cmd/assetsvc/pkg/utils/postgres_db_test.go

```diff
@@ -374,14 +374,11 @@ func TestGetPaginatedChartList(t *testing.T) {
 				}
 			}
 
-			charts, numPages, err := pam.GetPaginatedChartListWithFilters(ChartQuery{Namespace: tc.namespace, Repos: []string{tc.repo}}, 1, 10)
+			charts, err := pam.GetPaginatedChartListWithFilters(ChartQuery{Namespace: tc.namespace, Repos: []string{tc.repo}}, 0, 10)
```

Comment:
I've removed the `numPages` part of the return for the function since it isn't currently used in real life anyway (I assume it was to be able to present `page 1 of 10` or similar type pagination, which we do not do.

Reply:
Yep, that's precisely the reason why I added it. Right now, our UI is not displaying it, but, I still think letting the user know how many pages they is expected to fetch (thinking of a progress bar, or a more sophisticated scrolling like google photos ([interesting read here](https://medium.com/google-design/google-photos-45b714dfbed1)).
Without this value, we would have to calculate that number on the client's side, no?

Anyway, I'm ok with it as we don't have any real use case right now.